addons/google_calendar/models/res_users.py

```diff
@@ -71,7 +71,7 @@ def _sync_google_calendar(self, calendar_service: GoogleCalendarService):
         recurrences_write_dates = {r.id: r.write_date for r in odoo_recurrences}
         events_write_dates = {e.id: e.write_date for e in odoo_events}
         synced_recurrences = self.env['calendar.recurrence'].with_context(write_dates=recurrences_write_dates)._sync_google2odoo(recurrences)
-        synced_events = self.env['calendar.event'].with_context(write_dates=events_write_dates)._sync_google2odoo(events - recurrences, default_reminders=default_reminders)
+        synced_events = self.env['calendar.event'].with_context(write_dates=events_write_dates, from_google=True)._sync_google2odoo(events - recurrences, default_reminders=default_reminders)
```

Comment:
I would add this context in _write_from_google. Would it be possible?
That way if at some point we create an action to only sync one way, it would not break.

```py
    def _write_from_google(self, gevent, vals):
        self.with_context(from_google=True).write(vals)

    @api.model
    def _create_from_google(self, gevents, vals_list):
        return self.with_context(from_google=True).create(vals_list)

```